src/index.js

```diff
@@ -5,72 +5,135 @@ const addButton = document.getElementById('button');
 const todoList = document.getElementById('todo-list');
 const completeButton = document.querySelector('.complete');
 
-// add task
-document.addEventListener('DOMContentLoaded', () => {
-  const removeTask = (event) => {
-    const li = event.target.parentNode;
-    todoList.removeChild(li);
-  };
+const savedTasks = JSON.parse(localStorage.getItem('tasks')) || [];
 
-  const editTask = (event) => {
-    const li = event.target.parentNode;
-    const taskValue = li.firstChild.textContent.trim();
-    const newTaskValue = prompt('Edit task:', taskValue);
-    if (newTaskValue !== null && newTaskValue.trim() !== '') {
-      li.firstChild.textContent = newTaskValue.trim();
-    } else {
-      return;
-    }
-
-    const popupDiv = document.createElement('div');
-    popupDiv.classList.add('popup');
-    const heading = document.createElement('h2');
-    heading.textContent = 'Edit task';
-    const input = document.createElement('input');
-    input.type = 'text';
-    input.value = newTaskValue.trim();
-    const saveButton = document.createElement('button');
-    saveButton.textContent = 'Save';
-    saveButton.addEventListener('click', () => {
-      const editedTaskValue = input.value.trim();
-      if (editedTaskValue !== '') {
-        li.firstChild.textContent = editedTaskValue;
-        popupDiv.remove();
-      }
-    });
-    popupDiv.appendChild(heading);
-    popupDiv.appendChild(input);
-    popupDiv.appendChild(saveButton);
-    document.body.appendChild(popupDiv);
-  };
-
-  const addTask = () => {
-    const taskValue = taskInput.value.trim();
+/* eslint-disable */
 
```

Comment:
- [ ] It would be very nice :blush: if you don't disable linter and manually fix all the errors raised by linters because linters prevent you from making those errors in your code which can break your whole application :fearful: